docs/source/models/causes/neonatal/preterm_birth.rst

```diff
@@ -173,18 +198,22 @@ calculations.
       - Definition
       - Value or source
       - Note
-    * - mr_total
-      - neonatal mortality risk per live birth
-      - The mortality risk from the :ref:`Overall Neonatal Disorders Model <2021_cause_neonatal_disorders_mncnh>`
-      - The value of mr is a probability in [0,1]. Denominator includes live births only.
-    * - cause-specific mortality fraction
-      - fraction of all neonatal deaths due to neonatal preterm birth
-      - deaths_c381 / deaths_c380
-      -
-    * - frac_rds
+    * - :math:`\text{CSMR}`
+      - cause-specific mortality rate of preterm birth complications
+      - csmr_c381
+      - from GBD (CodCorrect)
+    * - :math:`f_k`
       - fraction of preterm deaths with RDS
       - XX%
```

Comment:
If we want to put a stand-in value for the engineers, I estimated that f_k is about 85% based on Table 4 in [this paper](https://www.sciencedirect.com/science/article/pii/S2214109X19302207).

Reply:
Actually, this definition is inconsistent with the usage in the formula above. To make it consistent with the above formula, you should should rename this variable $f\_\text{preterm with RDS}$, and add a new row called $f\_\text{preterm without RDS}$, which is defined as $1 - f\_\text{preterm with RDS}$.